tool/tsh/tsh.go

```diff
@@ -270,6 +282,9 @@ type CLIConf struct {
 	AWSRole string
 	// AWSCommandArgs contains arguments that will be forwarded to AWS CLI binary.
 	AWSCommandArgs []string
+
+	// ConfOptions is configuration read from the .tsh/config.yaml file.
+	ExtraProxyHeaders []ExtraProxyHeaders
```

Comment:
I'm a bit confused, why do we mention `ConfOptions` here? From what I can see it's just a local variable defined within `Run`. 🤔 I suppose it just might be an outdated comment as it also mentions `.tsh/config.yaml` but you say `.tsh/config/config.yaml` in the PR description.

Perhaps it should say that ExtraProxyHeaders are loaded from `./tsh/config/config.yaml`?

Reply:
Ah yeah I never updated some comments, whoops